src/app/user-intent/user-intent-view/user-intent-view.component.html

```diff
@@ -0,0 +1,12 @@
+<div class="container">
+    <div class="btn-group" dropdown>
+        <button dropdownToggle type="button" class="btn btn-primary dropdown-toggle">
+            Select Ecosystem <span class="caret"></span>
+        </button>
+        <ul *dropdownMenu class="dropdown-menu"  role="menu">
+            <li role="menuitem"><a class="dropdown-item" (click)="selectedEcosystem('Maven')">Maven</a></li>
```

Comment:
i would suggest we use lowecase as the value for each option. what do you say ?

Reply:
yes agree